data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+	on enter "Peacock"
+		dialog `"If correct, my estimates were, the most likely meeting point, this should be," Sedlitaris says as you jump into the system. She lets the pylon work for a while, but it returns no signal of a similar device. "Well... either estimated wrong, I have, or issues in meeting here, the two groups had. Check the next system, we should."`
+	on enter "Kaus Australis"
+		dialog `After a couple of minutes, the pylon emits a few quiet whistles, and Sedlitaris celebrates briefly, "On that planet, it is! Glad that we managed to locate it, I am." You tell her you'll land on Hopper soon, so that you can search for the pylon on the planet.`
+	to complete
+		never
+	to fail
+		has "Coalition: Expeditions Past 9: offered"
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 9"
+	landing
+	name "Escape Route"
+	description "Travel southwest of <system>, on a route to the Ablub system, so that Sedlitaris may look for a signal frequency of the expedition."
+	passengers 1
+	to offer
+		has "Coalition: Expeditions Past 8: active"
+	source "Hopper"
+	on offer
+		conversation
+			`You fly around <origin> as Sedlitaris searches for the pylon's exact position, in the hopes that she can bring it back as well. She tells you to land by a mountain range, and has you readjust your landing spot every so often as she tries to nail the exact location of the pylon. "Buried it to protect it from the atmosphere's acidity, they must have," she comments.`
+			`	By the time she's done, you open your ship's hatch right to where the pylon should be and help her dig it out, taking care to not be out for too long. When the device is dug out, you bring it inside your ship while Sedlitaris fills up the hole with what dirt had been dug out.`
+			`	As you're flying to the hangars, she plays the pylon's entry: "Log started. Made a route from Summit straight to the meeting point, I had, but find the split group, I could not. Restless, agitated, the crew has become, heading straight to Ablub, we are, after buried, this thing is. If find this, anyone from the split group is, head back-" at that point an untranslated shouting muffles what Mimodum was saying, followed by dozens of legs in a rush. "No..." is heard right before the recording ends.`
+			`	You focus on flying the ship, but even when you've landed, Sedlitaris remains silent.`
+			choice
+				`	"Are you alright?"`
+				`	"He said they were going back immediately, but they never made it. You think that was the last pylon?"`
+				`	(Stay quiet).`
+			`	She goes without a word for a while still. "Set a route for Ablub, could you, Captain? If follow the same route they used, we do, and if any more to find, there is, pick up the signals, I will."`
```

Comment:
Could be a bit difficult to keep this mission updated on the off-chance any new planets on the way to Ablub are added later on that would change the automatic course the game plots. Unlikely, but food for thought.

Reply:
At that point the mission would need updating to keep up, yeah.

Reply:
Since most of the game utilizes text replacements to make sure this isn't usually a problem, I'd think this could fall through the cracks if such a case occurs. I imagine we'd eventually discover the issue though. The only concern I have is if that route does change and Albub is no longer on it, then this whole ending might need to be revamped to accommodate that. I'm not too sure it's a good idea to merge with that in mind, but it's cool enough as-is that I'm torn. Gimme some time to chew on this one.

Reply:
If worse comes to worse, I suppose we could add some line later on mentioning the specific route included Albub - that'd be a pretty quick fix that wouldn't involve revamping the whole ending. With that in mind, I'm good with this.